distribution/changelog.txt

```diff
@@ -1,14 +1,17 @@
 0.3.2+ (in development)
 ------------------------------------------------------------------------
 - Feature: [#13057] Make GameAction flags accessible by plugins.
+- Feature: [#13078] [Plugin] Add colour picker widget.
 - Feature: [#13376] Open custom window at specified tab.
 - Feature: [#13398] Add pause button to the Track Designer.
+- Feature: [#13495] [Plugin] Add properties for park value, guests and company value.
 - Change: [#13346] Change FootpathScenery to FootpathAddition in all occurrences.
 - Fix: [#12895] Mechanics are called to repair rides that have already been fixed.
 - Fix: [#13257] Rides that are exactly the minimum objective length are not counted.
 - Fix: [#13334] Uninitialised variables in CustomTabDesc.
 - Fix: [#13342] Rename tabChange to onTabChange in WindowDesc interface.
 - Fix: [#13427] Newly created Go-Karts show "Race won by <blank>".
+- Fix: [#13431] [Plugin] UI disabled widgets can still be interacted with.
```

Comment:
should you perhaps add the plug in tag on the other existing fixes?

Reply:
🤔

Reply:
All changes to changelog in this PR have `[Plugin]`, so changing others can be done after this is merged.